### PR TITLE
💅 Very Empowered PR 💅

### PR DESCRIPTION
**Pull request by dahaca**
**Suite Empower is about to become canon!** 

For extra safety, this PR includes a copy of a _rebased and slightly patched_ latest version of the `suite-e` branch. At the moment of creation of this pull request, there is only [1 remaining suite-e PR](https://github.com/trezor/trezor-suite/pull/10855) and it should be merged to `develop` after this is merged.

### CODE REVIEW

**Comment by dahaca** [comment]
And of course there are some last minute bumps... 💀 

image

**Review by peter-sanderson — APPROVED** [review]

**Review by AdamSchinzel — APPROVED** [review]
The Eagle has landed 🦅

File: suite-common/wallet-core/src/accounts/accountsReducer.ts [hunk 1]
```diff
@@ -196,7 +196,10 @@ export const selectDeviceAccountsByNetworkSymbol = memoizeWithArgs(
 
         const accounts = selectDeviceAccounts(state);
 
-        return A.filter(accounts, account => account.symbol === networkSymbol);
+        return A.filter(
+            accounts,
+            account => account.symbol === networkSymbol && (!account.empty || account.visible),
```
Comment (Nodonisko) [inline suite-common/wallet-core/src/accounts/accountsReducer.ts:201]: I don't think this is good approach, if you need some specific derivation of Accounts (like `!empty || visible`) please create selector just for that. We will need to select empty accounts.

**Review by Nodonisko — CHANGES_REQUESTED** [review]

**Review by Nodonisko — APPROVED** [review]
LGTM after #10869

**Comment by peter-sanderson** [comment]
Here is another rebased branch https://github.com/trezor/trezor-suite/pull/10874

**Pull request closed without merging**